engine/src/main/java/org/hibernate/validator/internal/engine/cascading/ValueExtractorManager.java

```diff
@@ -31,6 +32,34 @@
 public class ValueExtractorManager {
 
 	@Immutable
+	public static final Set<ValueExtractor<?>> SPEC_DEFINED_EXTRACTORS = Stream.of(
+			ByteArrayValueExtractor.DESCRIPTOR,
+			ShortArrayValueExtractor.DESCRIPTOR,
+			IntArrayValueExtractor.DESCRIPTOR,
+			LongArrayValueExtractor.DESCRIPTOR,
+			FloatArrayValueExtractor.DESCRIPTOR,
+			DoubleArrayValueExtractor.DESCRIPTOR,
+			CharArrayValueExtractor.DESCRIPTOR,
+			BooleanArrayValueExtractor.DESCRIPTOR,
+			ObjectArrayValueExtractor.DESCRIPTOR,
+
+			LegacyListValueExtractor.DESCRIPTOR,
+			ListValueExtractor.DESCRIPTOR,
+
+			LegacyMapValueExtractor.DESCRIPTOR,
+			MapValueExtractor.DESCRIPTOR,
+			MapKeyExtractor.DESCRIPTOR,
+
+			LegacyIterableValueExtractor.DESCRIPTOR,
+			IterableValueExtractor.DESCRIPTOR,
+
+			OptionalValueExtractor.DESCRIPTOR,
+			ObjectValueExtractor.DESCRIPTOR
```

Comment:
So you don't include `ObservableValueValueExtractor` here? I can understand that it's a tough choice :).